modules/accessing-metrics-outside-cluster.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+[id="accessing-metrics-from-outside-cluster_{context}"]
+= Accessing metrics from outside the cluster for custom applications
+
+Learn how to query Prometheus statistics from the command line when monitoring your own services. You can access monitoring data from outside the cluster with the `thanos-querier` route.
+
+.Prerequisites
+
+* You deployed your own service, following the _Enabling monitoring for user-defined projects_ procedure.
+
+.Procedure
+
+. Extract a token to connect to Prometheus:
++
+[source,terminal]
+----
+$ SECRET=`oc get secret -n openshift-user-workload-monitoring | grep  prometheus-user-workload-token | head -n 1 | awk '{print $1 }'`
+----
++
+[source,terminal]
+----
+$ export TOKEN=`echo $(oc get secret $SECRET -n openshift-user-workload-monitoring -o json | jq -r '.data.token') | base64 -d`
+----
+
+. Extract your route host:
++
+[source,terminal]
+----
+$ export THANOS_QUERIER_HOST=`oc get route thanos-querier -n openshift-monitoring -o json | jq -r '.spec.host'`
+----
+
+. Query the metrics of your own services in the command line. For example:
++
+[source,terminal]
+----
+$ curl -X GET -k  'https://$THANOS_QUERIER_HOST/api/v1/query?query=up%7Bnamespace%3D%22ns1%22%7D' -H "Authorization: Bearer $TOKEN"
```

Comment:
This assumes there is a namespace called `ns1`. It would be good to mention it in this document.

Maybe we should change it to:
```
$ export NAMESPACE=ns1
$ curl -X GET -k 'https://$THANOS_QUERIER_HOST/api/v1/query?query=up%7Bnamespace%3D%22$NAMESPACE%22%7D' -H "Authorization: Bearer $TOKEN"